sass/_themes.scss

```diff
@@ -810,22 +814,26 @@ $themes: (
 
 :export {
     red: $color-red;
+    chart_baseline_up: $CHART_BASELINE_UP;
+    chart_baseline_down: $CHART_BASELINE_DOWN;
```

Comment:
As this is supposed to be the name of the closed graph, I think it would better to rename it to a more meaningful name. Furthermore, If we are going to determine the colors of `Area` chart, its related name is `ChartMountainBg` and `ChartMountainBorder`, and if we are supposed to add a disable color for other types of graph, I think we should use their colors.